resources/views/entities/pages/inventory/_form.blade.php

```diff
@@ -42,19 +45,12 @@
     <x-forms.field
         field="position"
         :label="__('entities/inventories.fields.position')">
-        {!! Form::text('position', null, [
-            'placeholder' => __('entities/inventories.placeholders.position'),
-            'class' => '',
+        {!! Form::select('position', $positionOptions, null, [
+            'data-placeholder' => __('entities/inventories.placeholders.position'),
+            'class' => 'position-dropdown',
             'maxlength' => 191,
             'list' => 'position-list',
```

Comment:
list is no longer needed. Does maxlength work here? I've never seen maxlength on a select field, but we must make sure user don't write a position with 200 letters that custs off